torchgeo/datasets/agb_live_woody_density.py

```diff
@@ -105,7 +105,7 @@ def _verify(self) -> None:
         # Check if the user requested to download the dataset
         if not self.download:
             raise RuntimeError(
-                f"Dataset not found in `root={self.paths}` and `download=False`, "
+                f"Dataset not found in `root={self.paths!r}` and `download=False`, "
```

Comment:
```suggestion
                f"Dataset not found in `paths={self.paths!r}` and `download=False`, "
```
We forgot to update this in the root -> paths PR, may need to be fixed in multiple files

Reply:
Noted, will look into this.